tutor/commands/compose.py

```diff
@@ -79,7 +81,7 @@ def update_docker_compose_tmp(
             docker_compose_jobs_tmp_path,
```

Comment:
Didn't you mention on an issue or PR somewhere that we could phase out the `*.jobs.yml` compose files at some point? Was it contingent on some refactoring?

If not, I think `*.task.yml` and `COMPOSE_LOCAL_TASKS_TMP` would be a more appropriate names for the compose file and filter. I don't want to block this PR any longer, so I'm happy to make a backlog ticket for that if you agree.

Reply:
Not so much an issue, more of an open question that I raised here: https://github.com/overhangio/tutor/issues/642
I don't want to phase out or rename this template or the corresponding patches at this point, as it would break plugins -- I'd rather not break plugins just for a name change. The rename would affect the following files:

- tutor/templates/k8s/jobs.yml
- tutor/templates/dev/docker-compose.jobs.yml
- tutor/templates/local/docker-compose.jobs.yml

And patches:

- local-docker-compose-jobs-services
- dev-docker-compose-jobs-services
- k8s-jobs

I don't think that the name change warrants a new backlog ticket; "jobs" might not be the perfect name but it's close enough.

Reply:
Sounds good.